src/System.Private.CoreLib/shared/System/Buffers/Text/Utf8Parser/ParserHelpers.cs

```diff
@@ -70,5 +70,29 @@ public static bool TryParseThrowFormatException<T>(out T value, out int bytesCon
             value = default;
             return TryParseThrowFormatException(out bytesConsumed);
         }
+
+        public static bool TryGetNextTwoDigits(ReadOnlySpan<byte> source, int prevSourceIndex, out int sourceIndex, out int value)
```

Comment:
Consider passing a sliced span instead. It is difficult to follow `prevSourceIndex`/`sourceIndex`
Alternatively, pass `sourceIndex` by ref (instead of having both `prevSourceIndex` and having an out).